.github/workflows/run_build.yaml

```diff
@@ -0,0 +1,56 @@
+name: run_build
+on: 
+  push:
+    branches:
+      - M2-6353-Develop-backend-workflow
```

Comment:
Did you mean to limit it to this branch?

Reply:
fixed, changed to push/pull_request for all branches